doc/source/user_guide/ug_implementation.rst

```diff
@@ -133,8 +133,10 @@ The user has several ways to initialize the grid, which can be read from
 files or created internally. The *rectgrid* code creates a regular rectangular 
 grid (use the namelist option ``grid_type='rectangular'``). The *popgrid* and *popgrid_nc* 
 code reads grid lengths and other parameters for a nonuniform grid (including tripole
-and regional grids). 
-The input files **grid_gx3.bin** and **kmt_gx3.bin** contain the
+and regional grids). The *mom_mosaic* code reads grids for a nonuniform grid defined 
+in the mom mosaic (supergrid) format.
```

Comment:
was there an objection in https://github.com/CICE-Consortium/CICE/pull/993 to the term "mosaic"?

Reply:
Oops thanks